.circleci/config.yml

```diff
@@ -163,6 +163,15 @@ jobs:
               conda config --add channels conda-forge
               conda activate curr_py
               conda install numba -q -y
+      - run:
```

Comment:
Need this for windows installation. In the PR to add prophet to automl we should also include installation instructions in our docs.